interoperator/pkg/controller/schedulers/sfdefaultscheduler/sfdefaultscheduler_controller.go

```diff
@@ -100,7 +105,7 @@ func (r *ReconcileSFDefaultScheduler) Reconcile(request reconcile.Request) (reco
 		return reconcile.Result{}, err
 	}
 	if instance.Spec.ClusterID == "" {
-		instance.Spec.ClusterID = "1"
+		instance.Spec.ClusterID = constants.MasterClusterID
```

Comment:
Shouldn't the environment variable for own cluster be used here first? `MasterClusterID` should be used only as default.

Reply:
The current assumption that master cluster has id "`1", will be addressed as a separate PR